docs/start/hello-world.ipynb

```diff
@@ -190,42 +202,101 @@
-      "qiskit_runtime_service.run:INFO:2024-03-11 20:15:30,537: Instance selected: ibm-q-internal/live-data/live-data-demo\n"
+      ">>> Job ID: crbfya9djmqg008kd5p0\n"
      ]
     }
    ],
    "source": [
     "# Construct the Estimator instance we want to use.\n",
+    "from qiskit_ibm_runtime import EstimatorV2 as Estimator\n",
     "\n",
-    "estimator = Estimator(backend)\n",
+    "estimator = Estimator(backend=backend)\n",
     "estimator.options.resilience_level = 1\n",
     "estimator.options.default_shots = 5000\n",
     "\n",
-    "observables = [\n",
+    "mapped_observables = [\n",
     "    observable.apply_layout(isa_circuit.layout) for observable in observables\n",
     "]\n",
     "\n",
     "# One pub, with one circuit to run against five different observables.\n",
-    "job = estimator.run([(isa_circuit, observables)])\n",
+    "job = estimator.run([(isa_circuit, mapped_observables)])\n",
+    "\n",
+    "# Use the job id in order to retrieve your job data at a later time\n",
+    "print(f\">>> Job ID: {job.job_id()}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "792d2f01",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This is the result of the entire submission.  We submitted one Pub,\n",
+    "# so this contains one inner result (and some metadata of its own).\n",
+    "job_result = job.result()\n",
+    "\n",
+    "# This is the result from our single pub, which had five observables,\n",
+    "# so contains information on all five.\n",
+    "pub_result = job.result()[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "48317a25",
+   "metadata": {},
+   "source": [
+    "<Admonition type=\"note\" title=\"Important\">\n",
+    "\n",
+    "Often times you will find that jobs will sit in the queue for some time.  If you'd like to quickly get results back for this small example circuit, use the code block provided below which utilizes the [`fake_provider`](../api/qiskit-ibm-runtime/fake_provider) submodule.\n",
+    "\n",
+    "</Admonition>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "id": "057f38f1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use the following code instead if you want to run on a simulator:\n",
+    "\n",
+    "\n",
+    "from qiskit_ibm_runtime.fake_provider import FakeAlmadenV2\n",
+    "backend = FakeAlmadenV2()\n",
+    "\n",
+    "estimator = Estimator(backend)\n",
+    "\n",
+    "\n",
+    "# Convert to an ISA circuit and layout-mapped observables.\n",
+    "pm = generate_preset_pass_manager(backend=backend, optimization_level=1)\n",
+    "isa_circuit = pm.run(qc)\n",
+    "\n",
+    "mapped_observables = [\n",
+    "    observable.apply_layout(isa_circuit.layout) for observable in observables\n",
+    "]\n",
+    "\n",
+    "job = estimator.run([(isa_circuit, mapped_observables)])\n",
+    "result = job.result()\n",
+    "\n",
```

Comment:
Thanks for providing full code for the simulator run! It might be cool to use tabs here, one for device run and the other for the simulator run. You can use the `<Tab>` component across cells.

Reply:
Ah good idea yeah!  I'll add that in.

Reply:
Actually in thinking about it, this would make the code cells unexecutable so CI wouldn't catch any problems and if a user downloads the notebook it would just be stored as some markdown and wouldn't be able to run it without modifying the notebook.

Reply:
I haven't tried it, but in recent discussions over converting tabs to notebooks, @frankharkins posted a workaround: https://github.com/Qiskit/documentation/issues/1143#issuecomment-2043202737 - not sure if it would help out here?

Reply:
Except that in his output, there are no tabs :)

Reply:
Haha right.... nvm

Reply:
The tabs don't show in the rendered markdown (sorry), but they are there in the source and render on the website https://qiskit-docs-preview-pr-1160.1799mxdls7qz.us-south.codeengine.appdomain.cloud/verify/plot-quantum-states

Reply:
And yes I was imagining we'd wrap the code cells in the tab component

Reply:
Yes I see it in the preview!